src/shared/components/Layout/UserAvatar.tsx

```diff
@@ -0,0 +1,23 @@
+import { Avatar, Menu, MenuButton, MenuItem, MenuList } from "@chakra-ui/react";
+import { signIn, signOut } from "next-auth/client";
+import React from "react";
+import { WithMaybeSession } from "types";
+
+export const UserAvatar: React.FC<WithMaybeSession> = ({ session }) => {
+  return (
```

Comment:
Looking at this running locally I think its a bit clearer if we show a 'sign in' button instead of a fallback avatar and then remove the sign-in option from the menu here. Happy to pop this in a different PR though this one is already a bit of a chonker

Reply:
This PR is getting chonky! I'll raise another issue!